src/displayBackend/wayland/WaylandZCopy.hpp

```diff
@@ -142,6 +143,45 @@ class WaylandKms : public WaylandZCopy
 
 typedef std::shared_ptr<WaylandKms> WaylandKmsPtr;
 
+/***************************************************************************//**
+ * Wayland Linux dmabuf class.
+ * @ingroup wayland
+ ******************************************************************************/
+class WaylandLinuxDmabuf : public WaylandZCopy
+{
+public:
+
+	~WaylandLinuxDmabuf();
+
+	DisplayItf::FrameBufferPtr createLinuxDmabufBuffer(
+			DisplayItf::DisplayBufferPtr displayBuffer,
+			uint32_t width,uint32_t height, uint32_t pixelFormat);
+
+private:
+
+	zwp_linux_dmabuf_v1* mWlLinuxDmabuf;
+	zwp_linux_dmabuf_v1_listener mWlListener;
+
+	friend class Display;
+
+	WaylandLinuxDmabuf(wl_registry* registry, uint32_t id, uint32_t version);
+
+	static void sOnModifiers(void *data,
+							 zwp_linux_dmabuf_v1 *zwp_linux_dmabuf,
+							 uint32_t format, uint32_t modifier_hi,
+							 uint32_t modifier_lo);
+	static void sOnFormat(void *data,
+						  zwp_linux_dmabuf_v1 *zwp_linux_dmabuf,
+						  uint32_t format);
+
+	virtual void authenticate() override {};
+
+	void init(uint32_t version);
+	void release();
+};
```

Comment:
Is it supposed to use: copy ctor/move ctor/operator= ?
If not, please add the corresponding declarations.

Reply:
I am not sure I am following you here. @al1img any comments on this?

Reply:
if it is not supposed to use, you could do something like here:
https://github.com/xen-troops/camera_be/blob/master/src/MediaController.hpp#L23